README.NewResourceFromTerraform.md

```diff
@@ -197,6 +197,34 @@ ServiceMappings file. Add the `ResourceConfig` for your resource:
         - remove_default_node_pool
     ```
 
+1. Add `mutableButUnreadableFields` if necessary. Fields may not be existent in the underlying GCP resource,
```

Comment:
Instead of trying to explain it, let's just give an example.

For example, a password field is mutable because users may update their password through the API by providing a new value. However a password is not readable because a GET call of the corresponding GCP resource usually will not return the password value as a plaintext string for security considerations.